.circleci/config.yml

```diff
@@ -1,44 +1,59 @@
 workflows:
```

Comment:
This doesn't look intended, please revert by running:

```shell
git checkout upstream/develop .circleci/config.yml
```

Reply:
Yaah,I saw this file being changed after pushing commit.

Reply:
@Lokesh2703 it still should not appear. I'd recommend copy and pasting it from the repo and committing this again.

Reply:
@kevinlee12 when I try to copy and paste from repo, its is showing that there are no changes from the code in repo with the code present in this commit. And in the commit section, it is showing the commit which is changing .circleci/config.yml is made by kelvinlee12.

Reply:
go ahead and create another PR, I'll go ahead and close this. thanks!